src/components/local-weather.jsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { fetchWeather } from "../utils";
+import Search from "./search";
+import Today from "./today-weather";
+import FiveDay from "./five-day-weather";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+
+class LocalWeather extends React.Component {
+  constructor(props) {
```

Comment:
Imagine that this app was large enough to require some form of global state management. How would you go about integrating a solution?

Reply:
If this was a larger app, I would use a global React context to keep track of the current city & its weather data, and a reducer that would replace the current state. I would also cache information like a location’s city name and lat/long so that we wouldn’t have to do two calls for each city look up if the user has already looked up the city once.